pkg/ovs/ovsctl/appctl.go

```diff
@@ -227,3 +228,45 @@ func (c *ovsCtlClient) GetDPFeatures() (map[DPFeature]bool, error) {
 	}
 	return features, nil
 }
+
+func (c *ovsCtlClient) DeleteDPInterface(name string) error {
```

Comment:
Probably add a comment saying if the interface does not exists, the func just returns with no error.